crates/logutil/src/lib.rs

```diff
@@ -84,8 +86,17 @@ pub fn init(verbosity: impl Into<Verbosity>, mode: LoggingMode) {
             subscriber::set_global_default(subscriber)
         }
         LoggingMode::Full => {
-            let subscriber = full_fmt(level).with_env_filter(env_filter).finish();
-            subscriber::set_global_default(subscriber)
+            let subscriber = full_fmt(level).with_env_filter(env_filter);
+
+            if let Some(file) = log_file {
+                let debug_log = {
+                    let file = File::create(file).expect("Failed to create log file");
```

Comment:
Can we instead just `eprintln` a warning here, instead of a panic!

Reply:
`eprintln` will typically let the execution continue, but there won't be any file to write the logs to.